client/state/ui/guided-tours/contexts.js

```diff
@@ -140,7 +140,8 @@ export const hasSelectedSiteDefaultSiteTitle = state => {
  * @return {Boolean} True if user has just pasted something from Google Docs, false otherwise.
  */
 export const hasUserPastedFromGoogleDocs = state => {
-	return getLastAction( state ).type === EDITOR_PASTE_EVENT_FROM_GDOCS;
+	const action = getLastAction( state ) || false;
+	return action && ( action.type === EDITOR_PASTE_EVENT_FROM_GDOCS );
```

Comment:
Both statements here are defensive, you don't need both at the same time 😄 

What I meant in my other comment was that `getLastAction` itself could be changed to fall back to `false`, but in this case here I'd choose to keep only one: either `|| false` on the first line or `action &&` on the second. The latter is more conventional.